tests/unit/test_middleware/test_rate_limit_middleware.py

```diff
@@ -28,33 +28,46 @@ async def test_rate_limiting(unit: DurationUnit) -> None:
     def handler() -> None:
         return None
 
-    config = RateLimitConfig(rate_limit=(unit, 1))
+    config = RateLimitConfig(rate_limit=(unit, 2))
     cache_key = "RateLimitMiddleware::testclient"
     app = Litestar(route_handlers=[handler], middleware=[config.middleware])
     store = app.stores.get("rate_limit")
 
     with travel(datetime.utcnow, tick=False) as frozen_time, TestClient(app=app) as client:
         response = client.get("/")
-        assert response.status_code == HTTP_200_OK
+
         cached_value = await store.get(cache_key)
         assert cached_value
         cache_object = CacheObject(**decode_json(value=cached_value))
         assert len(cache_object.history) == 1
```

Comment:
IMO, this block code is unnecessary because it tests implementation details. WDYT?

Reply:
While it does test implementation details, that's intentional here, since it ensure e.g. that we don't accidentally save more things in the history than we intend to